elastica/contact_forces.py

```diff
@@ -0,0 +1,77 @@
+__doc__ = """ Numba implementation module containing contact between rods and rigid bodies and other rods rigid bodies or surfaces."""
+
+from elastica.typing import SystemType, AllowedContactType
+from elastica.rod import RodBase
+
+
+class NoContact:
+    """
+    This is the base class for contact applied between rod-like objects and allowed contact objects.
+
+    Notes
+    -----
+    Every new contact class must be derived
+    from NoContact class.
+
+    """
+
+    def __init__(self):
+        """
+
+        Parameters
+        ----------
+
```

Comment:
why there if no params?

Reply:
Me and @armantekinalp had some talks about whether we should make it similar to FreeJoint where it does some basic contact (given nu and k) or should it be like NoForces which does not need any input parameters. We decided to make it like NoForces in the end but I forgot to remove this, so I will remove it in the next commit.